### ISSUE TITLE
Update Mailchimp API to 2.4

### ISSUE THREAD

**Issue by nabossha** [sup7even/mailchimp#18]
it would be nice having an update of the included Mailchimp-API to have the ability for passing  the subscribers language to Mailchimp.

see: https://github.com/drewm/mailchimp-api/commit/b04fa5fd6b89a2dbbad0d587e537827e439f8ad8#diff-4658eb423d377d316ca4cdeb41f5c5da

after updating the API the subscribers language can be passed to Mailchimp by using this code within the MailchimpHook-class:

`$params['data']['language'] = $GLOBALS['TSFE']->config['config']['language'];`

**Comment by liayn** [comment]
This should be done together with #30 maybe?

**Comment by Tuurlijk** [comment]
@liayn No, please do a release first. Then a minor update for the new api usage. I tried, but it's not a drop in replacement.
